playbooks/roles/ocp-csi-driver/tasks/main.yml

```diff
@@ -117,13 +117,15 @@
       name: powervc-certificate
       namespace: default
     register: powervc_certificate
+    ignore_errors: yes
 
-  - name: Fail if the powervc-certificate does not exists in default namespace
-    fail:
+  - name: If the powervc-certificate does not exists in default namespace
+    debug:
       msg: "PowerVC certificate secret is not present!"
     when: powervc_certificate.resources | length == 0
 
-  - fail:
+  - name: If powervc-certificate is empty
```

Comment:
This task will fail if the above condition is true.

Reply:
I ran it on a cluster - it just gives a message
```
TASK [ocp-csi-driver : Check if the PowerVC certificate's secret is exists] *************************************************************************************************************************************************************************
ok: [localhost]

TASK [ocp-csi-driver : If the powervc-certificate does not exists in default namespace] *************************************************************************************************************************************************************
ok: [localhost] => {
    "msg": "PowerVC certificate secret is not present!"
}

TASK [ocp-csi-driver : If powervc-certificate is empty] *********************************************************************************************************************************************************************************************
ok: [localhost] => {
    "msg": "PowerVC certificate is empty! Please check the certificate's secret."
}
```

Reply:
I have removed the name parameters and replaces fail with debug

Reply:
```
TASK [ocp-csi-driver : Check if the PowerVC certificate's secret is exists] *************************************************************************************************************************************************************************
ok: [localhost]

TASK [ocp-csi-driver : If the powervc-certificate does not exists in default namespace] *************************************************************************************************************************************************************
ok: [localhost] => {
    "msg": "PowerVC certificate secret is not present!"
}

TASK [ocp-csi-driver : debug] ***********************************************************************************************************************************************************************************************************************
ok: [localhost] => {
    "msg": "PowerVC certificate is empty! Please check the certificate's secret."
}

TASK [ocp-csi-driver : Create namespace for CSI driver validation] **********************************************************************************************************************************************************************************
ok: [localhost]
```